bindings/c/test/mako/mako.cpp

```diff
@@ -2566,18 +2613,29 @@ int main(int argc, char* argv[]) {
 
 	if (proc_type == ProcKind::WORKER) {
 		/* worker process */
-		workerProcessMain(args, worker_id, shm_access, pid_main);
+		auto time_start = steady_clock::now();
+
+		workerProcessMain(args, process_idx, shm_access, pid_main);
+
+		shm_access.processStatsSlot(process_idx)
```

Comment:
I am wondering how significant is the CPU usage before starting the workload. Could you try printing out the time_start values of different threads if they match within 1ms range. Also check what is getProcessorTimeProcess the value at the beginning of the actual workload, I would say just before the main transaction loop and see if it negligible (e.g. <  0.1 s). Otherwise we should consider subtracting the CPU usage before the workload starts.

Reply:
I did some tests. It depends on the parameters, but I observed values > 0.1 s. For example, in the sync mode, with 2 databases, 2 clients per version, 128 threads and 1 process, the times measured right before the main loop were between 0.168 and 0.223. Increasing the number of threads to 1024 changes this range to  0.224 s -- 0.67 s.

The durations measured right after calling the `workerThread()` function with the first configuration were between 0.149 and 1.68.

Reply:
Then it would be good to improve the precision by starting the measurement (the start of both elapsed time and CPU counter) closer to the beginning of the actual workload.